src/tests/ftest/launch.py

```diff
@@ -747,12 +719,12 @@ def get_test_files(test_list, args, yaml_dir, vmd_flag=False):
     """
     # Replace any placeholders in the extra yaml file, if provided
     if args.extra_yaml:
-        args.extra_yaml = replace_yaml_file(args.extra_yaml, args, yaml_dir, vmd_flag)
+        args.extra_yaml = replace_yaml_file(args.extra_yaml, args, yaml_dir)
 
     test_files = [{"py": test, "yaml": None, "env": {}} for test in test_list]
     for test_file in test_files:
         base, _ = os.path.splitext(test_file["py"])
-        yaml_file = replace_yaml_file("{}.yaml".format(base), args, yaml_dir, vmd_flag)
+        yaml_file = replace_yaml_file("{}.yaml".format(base), args, yaml_dir)
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string